1.10.10/alpine3.10/include/pip-constraints.txt

```diff
@@ -12,3 +12,6 @@ WTforms<2.3.0
 
 # Details: https://github.com/apache/airflow/issues/8599
 flask-appbuilder<2.3.3
+
+# Avoid Version check if Version Check Table doesn't exist
+astronomer-airflow-version-check~=1.0.1
```

Comment:
This would only install 1.0.*, but not a 1.1.

Reply:
Right yes.